lib/qudaQKXTM_Kepler_utils.cpp

```diff
@@ -0,0 +1,1413 @@
+#include <cufft.h>
```

Comment:
I find it a very good idea to split the qudaQKXTM_Kepler.cpp into separate files. In this sense, I have the following suggestions:

- Create a new file lib/qudaQKXTM_Loop_Kepler.cpp. Move functions:
    - oneEndTrick_w_One_Der from qudaQKXTM_Kepler_utils.cpp:line-212 into qudaQKXTM_Loop_Kepler.cpp
    - writeLoops_ASCII from from qudaQKXTM_Kepler_utils.cpp:line-867 into qudaQKXTM_Loop_Kepler.cpp
    - getLoopWriteBuf from qudaQKXTM_Kepler_utils.cpp:line-941 into qudaQKXTM_Loop_Kepler.cpp
    - writeLoops_HDF5 from qudaQKXTM_Kepler_utils.cpp:line-967 into qudaQKXTM_Loop_Kepler.cpp
    - Loop_w_One_Der_FullOp_Exact from qudaQKXTM_Deflation_Kepler.cpp:line-1916 into qudaQKXTM_Loop_Kepler.cpp

With this, we also gather the loop-related functions into their own separate file, and qudaQKXTM_Kepler_utils.cpp is left with just the real "utility functions".